### DISCUSSION TITLE
Change wiki login page URL

### DISCUSSION THREAD

**Discussion by anandroczz** [requarks/wiki#5969] [Help / Questions]
Can we change wiki login page URL? For example : If the login page URL is http://10.10.10.10 and want to change it to 10.10.10.20 can this be done?
Should we un install and reinstall to achieve this or can we change it somewhere from the setup or config file?
If we can do it by changing it from the file, may I know in which file I must make the changes and the path of that particular file.

Wiki is installed in Linux and using Postgres as DB.

Can I get response asap?

**Comment by NGPixel** [comment]
I'm not sure what you're referring to with "login page URL". The login page is always at `/login` of whatever IP / hostname its installed on...

**Reply by anandroczz** [reply]
> I'm not sure what you're referring to with "login page URL". The login page is always at `/login` of whatever IP / hostname its installed on...

Server IP has been changed from X.X.X.X to Y.Y.Y.Y but now the wiki is not accessible with Y.Y.Y.Y and I don't remember the IP of X.X.X.X

**Reply by anandroczz** [reply]
> > I'm not sure what you're referring to with "login page URL". The login page is always at `/login` of whatever IP / hostname its installed on...
> 
> Server IP has been changed from X.X.X.X to Y.Y.Y.Y but now the wiki is not accessible with Y.Y.Y.Y and I don't remember the IP of X.X.X.X

Now is there a way to find X.X.X.X in any of the file from backend?

**Reply by NGPixel** [reply]
That's not how it works. Wiki.js doesn't bind to a specific IP not does it keep any IP configuration. It will be accessible from any IP or hostname pointing to that server.

Have you tried restarting wiki.js or the server itself?

**Reply by anandroczz** [reply]
> That's not how it works. Wiki.js doesn't bind to a specific IP not does it keep any IP configuration. It will be accessible from any IP or hostname pointing to that server.
> 
> Have you tried restarting wiki.js or the server itself?

May I know how to restart wiki.js. I've installed it on ubuntu using linux installation procedure. not the docker process.
